### PR TITLE
[Monit-27990] File buffer lock

### PR DESCRIPTION
**Pull request by laullon**

### CODE REVIEW

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+services:
+  proxy-1:
+    build: .
+    environment:
+      WAVEFRONT_URL: http://host.docker.internal:8080
+      WAVEFRONT_TOKEN: dhgjfdhgsjlkdf22340007-8fc6-4fc6-affa-b000ffa590ef
+      WAVEFRONT_PROXY_ARGS: --ephemeral false --idFile /var/spool/wavefront-proxy/id-1
+    volumes:
+      - /Users/glaullon/tmp:/var/spool/wavefront-proxy
```
Comment (joannak-vmware) [inline docker/docker-compose.yml:9]: Do we want this value hardcoded?

File: docker/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    ports:
+      - "2878:2878"
+  proxy-2:
+    build: .
+    environment:
+      WAVEFRONT_URL: http://host.docker.internal:8080
+      WAVEFRONT_TOKEN: dhgjfdhgsjlkdf22340007-8fc6-4fc6-affa-b000ffa590ef
+      WAVEFRONT_PROXY_ARGS: --ephemeral false --idFile /var/spool/wavefront-proxy/id-2
+    volumes:
+      - /Users/glaullon/tmp:/var/spool/wavefront-proxy
```
Comment (joannak-vmware) [inline docker/docker-compose.yml:19]: same here

**Review by joannak-vmware — APPROVED** [review]
Please check comments on `docker/docker-compose.yml`. Otherwise LGTM

File: docker/docker-compose.yml (hunk 1 above)
Comment (laullon) [reply docker/docker-compose.yml:9]: solved. anyway this is just for testing

**Pull request merged**